docs/getting-started/Installation.stories.mdx

```diff
@@ -43,6 +43,29 @@ To import Orbit experimental components styles, also add the following import de
 @import "@orbit-ui/experimental/index.css";
 ```
 
+## Set up your tooling environment
+
+Orbit offer an ESLint plugin to see in-context help in your IDE. This includes accessibility pointers, deprecation notices, and other helpful tips.
+
+First, add the ESlint plugin to your dependencies:
+
+<PackageInstallationSnippet workspaceFolder="tooling" packageName="eslint-plugin" />
+
+Second, add the plugin to your ESlint config. For example, your .eslintrc.json file may look like this:
+
+```json
+{
+    "$schema": "https://json.schemastore.org/eslintrc",
+    "plugins": ["@orbit-ui"],
+    "extends": [
+        // ...
```

Comment:
Personally, for the new documentation that I am writting, I dont add those "..." comment anymore as it doesn't works well with copy/paste.